source/nordic_sdk/components/ble/common/ble_srv_common.c

```diff
@@ -39,7 +39,7 @@
 #include <string.h>
 #include "nordic_common.h"
 #include "app_error.h"
-
+#include "ble.h"

```

Comment:
@rgrover In theory, all files including ble.h should only find ble.h as BLE.h is not exposed with extraIncludes. All files intending to use BLE.h would use ble/BLE.h. The problem only comes if a file, intending to use BLE.h, uses `#include BLE.h`, on mac os, this can find ble.h instead.